.npmignore

```diff
@@ -7,4 +7,5 @@ src/
 test/
 !lib/
 .github/
-coverage
\ No newline at end of file
+coverage
+.vscode/
```

Comment:
Should those be checked into Git at all? I'm not sure if it's ideal having IDE specific settings checked into a repo for open source projects.